app/javascript/redux/Slice.js

```diff
@@ -10,6 +10,7 @@ const initialState = {
 export const getMessageData = createAsyncThunk('message/getMessageData', async () => {
      const response = await fetch(API);
      const data = await response.json();
+     console.log(data);
      return data;

```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Kindly remove the console log to follow the JavaScript best practices :+1: and also it kills the performance of your application :fearful: